Detectors/EMCAL/reconstruction/src/CaloRawFitter.cxx

```diff
@@ -255,14 +255,14 @@ std::tuple<int, int, float, short, short, float, int, int> CaloRawFitter::preFit
   // something valid was found, and non-zero amplitude
   if (index >= 0 && maxamp >= acut) {
     // use more convenient numbering and possibly subtract pedestal
-    auto [ped, mReversed] = reverseAndSubtractPed((bunchvector.at(index)), altrocfg1, altrocfg2);
+    std::tie(ped, mReversed) = reverseAndSubtractPed((bunchvector.at(index)), altrocfg1, altrocfg2);
```

Comment:
Is there a reason to 1st fill the local array in ``reverseAndSubtractPed`` , then assign it to ``mReversed`` (extra loop) and then to loop again to find the max element? All this can be done in a single loop with direct assignment to mReversed

Reply:
Fixed. Now we have only one loop. Thanks for spotting this.